tests/debug_project/pyproject.toml

```diff
@@ -0,0 +1,13 @@
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[project]
+name = "debug"
```

Comment:
I'm overall not the best at naming things, so I welcome suggestions of what this test package should be called so that people understand that it exists only for testing and should ignore it. Though `test_package_ignore` could work.